### ISSUE TITLE
AWX multiple Choice does not work and unable to add  options 

### ISSUE THREAD

**Issue by Sreenadh8** [ansible/awx#4192]
##### ISSUE TYPE
 - Bug Report

Component Name

UI

##### SUMMARY

I am not able to add 2 options to the survey when I chose the answer type multiple options single select or multi select..

##### ENVIRONMENT
* AWX version: 3.0.1 , 4.0.0 and 5.0.0
* AWX install method: docker on linux,
* Ansible version:  2.8.1 , 2.7
* Operating System: Ubuntu 18.04
* Web Browser: Chrome, IE

##### STEPS TO REPRODUCE

Add one question to survey with mutliple Choice(Single Select or multi-select) type . And try to add options in 'choice options'...

##### EXPECTED RESULTS
Once I hit enter in the multiple choice options box, I am expecting to go to next line to add my second option.

##### ACTUAL RESULTS

it stuck in the first line

##### ADDITIONAL INFORMATION

I tried with AWX 3.0.1, 4.0.0 and even latest version 5.0 , still this issue exist... please help to fix this issue...
![awx multi-choice issue](https://user-images.githubusercontent.com/41424639/60383708-1e8d7300-9a92-11e9-865d-c2b8c77d3a32.JPG)

**Comment by fxfitz** [comment]
Some users on our team are experiencing this. Did you happen to find out a fix @Sreenadh8 ??

**Comment by AlexSCorey** [comment]
I have tried to reproduce this in awx and in Tower 3.3.3 and I have been unsuccessful. I can successfully create a textarea and multiselect survey question from new Job Templates and from existing job templates.

**Comment by ryanpetrello** [comment]
Can we move this to `needs_test`?  I've personally experienced this off and on in Google Chrome, and I think it _is_ a real bug in modern versions of AWX.

**Comment by dsesami** [comment]
This is reproducible on AWX 5.0.0 but has been fixed in AWX 7.0.0/latest. leaving next steps to you @AlexSCorey

**Comment by AlexSCorey** [comment]
@dsesami Thanks for checking into this.  In conversation with @ryanpetrello we are going to see if we get more reports, that offer more details as to the source of the problem, and move from there.

**Comment by fxfitz** [comment]
For what it's worth, we observed this on v6.0.0. Haven't upgraded to 7.0.0 yet.

**Comment by dsesami** [comment]
@fxfitz ~this was fixed _very_ recently, so I wouldn't be surprised if it was on 6.0.0. If you are able to upgrade, it should resolve it.~ see correction below.

**Comment by fxfitz** [comment]
Gotcha. We'd like to upgrade, but right now we're seeing problems with our role requirements downloading, so until that's fixed we won't be able to upgrade.

**Comment by ryanpetrello** [comment]
@dsesami what change to AWX do we suspect resolved this?

**Comment by dsesami** [comment]
@ryanpetrello @fxfitz after some sanity checks with different instances:

First, I completely misremembered and got this confused with something else. apparently this bug isn't fixed yet. For that miscommunication I apologize. https://github.com/ansible/awx/issues/4343 also exists about this.

Apparently this is something that can affect different textareas in the app, but it happens consistently on some AWX instances and not on others, so this is ongoing. @AlexSCorey has a repro he's going to try to dive into.

**Comment by rebeccahhh** [comment]
Per discussion with @AlexSCorey I'm adding my own two cents.
In addition to the existing bug (not being able to enter/newline in the UI) I seem to be having the newline issue in the API page post as well. 
When posting something along the lines of `{"choices": "test\ntest2\ntest3"}` (along with other required/associated JSON variables) on the API page and then returning to the UI, the \n is ignored and presents as one choice.

**Comment by mabashian** [comment]
I've got a PR up to address what I think is the root cause of this behavior. See: https://github.com/ansible/awx/pull/4869.  For those that do come across this before #4869 merges I believe a hard refresh of the page will alleviate the issue.

The easiest reproducer that I came across was:

1. Create a new personal access token (when you create the token, an Alert modal is shown with a key in it)
2. Without refreshing the page navigate to any textarea and attempt to hit enter.  Examples could be the inventory script form or survey questions in the launch modal.
